src/common/utils/index.ts

```diff
@@ -21,7 +21,9 @@
 
 // Common utils (main OR renderer)
 
-export const noop: any = () => { /* empty */ };
+export function noop<T extends any[]>(...args: T): void {
```

Comment:
Looks like unnecessary change? Is it?

Reply:
It is a minor improvement in the typing. That is all. I forget why I did this in the original PRs.